content/blog/living-off-the-ipad-as-an-engineer/index.md

```diff
@@ -0,0 +1,110 @@
+---
+{
+    title: 'Living off the iPad as an Engineer',
+    description: "Tips on how to get yourself a proper development environment on the iPad to fully exploit its potential.",
+    published: '2021-02-01T00:00:00.000Z',
+    authors: ['pierremtb'],
+    tags: ['tooling', 'ipad'],
+    attached: [],
+    license: 'cc-by-nc-nd-4'
+}
+---
+Since I transitioned from working all day on my personal MacBook Pro to receiving a work computer for a new engineering position, I decided to go for the only thing financially wise: selling my beloved 16" MacBook Pro to live off the 2018 iPad Pro I had around for iOS app development as **main computer** for my personal life.
+
+While I loved the feeling of knowing that I could open and run anything on the MacBook Pro—aka a conventional laptop—the idea of moving solely to the efficient machine that is the iPad Pro was appealing for various reasons. Yet the question remained: how would I continue the work on side-projects, whether they be software or hardware? We see lots of talking these days on how [LumaFusion](https://freecadweb.org/) is real competition to Adobe Premiere, or that [Affinity Photo](https://affinity.serif.com/en-gb/photo/ipad/) has nothing to envy from desktop Photoshop. While I do spend some time on such creative apps, how am I supposed to maintain [my personal webpage](https://pierrejacquier.com), write code for my Raspberry Pi, or create CAD models for 3D printing?
```

Comment:
While I loved the feeling of knowing that I could open and run anything on the MacBook Pro — aka a conventional laptop — the idea of moving solely to the efficient machine that is the iPad Pro was appealing for various reasons. Yet the question remained: how would I continue the work on side-projects, whether they be software or hardware? There is a lot of talk these days about how [LumaFusion](https://freecadweb.org/) is real competition to Adobe Premiere, or that [Affinity Photo](https://affinity.serif.com/en-gb/photo/ipad/) has nothing to fear from desktop Photoshop. While I do spend some time with such creative apps, how am I supposed to maintain [my personal webpage](https://pierrejacquier.com), write code for my Raspberry Pi, or create CAD models for 3D printing?

Reply:
Thanks for the changes they make a lot of sense. May I just ask why the em-dashes “—“ would takes spacing in your opinion? My understanding was that they don’t, as per https://www.thepunctuationguide.com/em-dash.html, but I could be wrong!

Reply:
You're correct about the em-dash rule but it's frequently ignored in digital publications (as opposed to in physical print, where I see it without spaces all the time). 

With digital text, things sometimes get weird depending on font choice and other CSS factors. Spaces give you a little insurance against the vagaries of digital fonts and web browser rendering engines.

It's really not a big deal either way. Feel free to take out the spaces if you feel more comfortable that way or just like the way it looks!

Reply:
Oh I see. That makes a lot of sense. Thanks for explaining!